samples/subsys/usb/testusb/sample.yaml

```diff
@@ -1,7 +1,7 @@
 sample:
-  name: USB loopback sample
+  name: USB device loopback sample
 tests:
-  sample.usb.loopback:
-    depends_on: usb_device
+  sample.usbd.loopback:
+    depends_on: usbd
     tags: usb
-    harness: button
+    harness: TBD
```

Comment:
This looks wrong.

Reply:
Change to `build_only`

Reply:
fwiw there's definitely a few other places in-tree where harness is `TBD`

Reply:
We should fix them as well.

Reply:
changed to `build_only: true`